dali/python/CMakeLists.txt

```
@@ -26,6 +26,15 @@ target_link_libraries(${dali_python_lib} PRIVATE ${CUDA_LIBRARIES} ${dali_lib})
 set_target_properties(${dali_python_lib} PROPERTIES
   LIBRARY_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/dali/python/nvidia/dali)
 
+string(TIMESTAMP TODAY "%Y%m%d")
+if (MAKE_NIGHTLY_BUILD)
+  set(DALI_FLAVOR "-nightly")
+  set(DALI_VERSION "${DALI_VERSION}${TODAY}")
+elseif (MAKE_WEEKLY_BUILD)
+  set(DALI_FLAVOR "-weekly")
+  set(DALI_VERSION "${DALI_VERSION}${TODAY}")
```

```suggestion
  set(DALI_VERSION "${DALI_VERSION}.${TODAY}")
```
Maybe some kind of delimiter between DALI version and the date stamp?